src/engine/Sprite.cxx

```diff
@@ -33,8 +34,9 @@ void Sprite::render() const
         SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, highlightColor.r, highlightColor.g, highlightColor.b);
       }
 
-      if (transparentSprite)
+      if (currentLayer == Layer::ZONE)
       {
+        unsigned char alpha = (1 - Settings::instance().zoneLayerTransperancy) * 255;
         SDL_SetTextureAlphaMod(m_SpriteData[currentLayer].texture, alpha);
```

Comment:
Use set alpha value:

```suggestion
SDL_SetTextureAlphaMod(m_SpriteData[currentLayer].texture, m_SpriteData[currentLayer].alpha);
```

Reply:
it was supposed to be like this, but here we take from Settings and not alpha
this is only for Zone Layer
it was intentional.